msu/files.py

```diff
@@ -1,16 +1,43 @@
 from urllib import parse
-from flask import current_app
 import boto3
 
+from flask import (
+    current_app,
+    flash,
+    request
+)
+
+
 s3_client = boto3.client('s3')
 
 def _url(bucket, loc_constraint, name):
     return 'https://%s.s3.%s.amazonaws.com/%s' % \
         (bucket, loc_constraint, name)
 
-def upload(key, data):
-    bucket = current_app.config['S3_BUCKET']
+def _upload(key, data, bucket_extension):
+
+    bucket = current_app.config[bucket_extension]
     constraint = s3_client.get_bucket_location(Bucket=bucket)['LocationConstraint']
     s3_client.upload_fileobj(data, bucket, key)
 
     return _url(bucket, constraint, key)
+
+def upload_file(key, data):
+    return _upload(key, data, 'S3_BUCKET_FILES')
+
+def upload_image(key,data):
+    return _upload(key, data, 'S3_BUCKET_IMAGES')
+    
+def check_file(request):
+    if 'file' not in request.files:
+            print("1")
+            flash('No file part')
+            return False
+
+    file = request.files['file']
+
+    if file.filename == '':
+        print("1")
+        flash('No file selected')
+        return False
+    return True
```

Comment:
I think your editor is misconfigured and is thus not adding a newline to the last line of the file. Not a big deal, but still worth fixing now for consistency.